eng/scripts/Language-Settings.ps1

```diff
@@ -23,16 +25,30 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
   {
     if (!$projectOutput) { continue }
 
-    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
-
+    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk, $dllFolder = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
     if(!(Test-Path $pkgPath)) {
       Write-Host "Parsed package path `$pkgPath` does not exist so skipping the package line '$projectOutput'."
       continue
     }
+
+    # Add a step to extract namespaces
+    $namespaces = @()
+    # The namespaces currently only use for docs.ms toc, which is necessary for internal release.
+    if (Test-Path "$dllFolder/Release/netstandard2.0/") {
+      $defaultDll = Get-ChildItem "$dllFolder/Release/netstandard2.0/*" -Filter "$pkgName.dll" -Recurse
+      if ($defaultDll -and (Test-Path $defaultDll)) {
+        Write-Host "Here is the dll file path: $($defaultDll.FullName)"
```

Comment:
Lets remove this logging or change it to verbose as we don't want this showing up everywhere we run.

Reply:
Sure.